src/style.css

```diff
@@ -0,0 +1,161 @@
+body {
+  margin: 0;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
```

Comment:
it's 100% by default, it was mentioned when your previous task was reviewed. Please, remember to apply previous comments here too